src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/SlimefunItem.java

```diff
@@ -430,6 +430,7 @@ private void loadItemHandlers() {
             if (!handler.isPrivate()) {
                 Set<ItemHandler> handlerset = getPublicItemHandlers(handler.getIdentifier());
                 handlerset.add(handler);
+                SlimefunPlugin.getRegistry().getPublicItemHandlers().replace(handler.getIdentifier(), handlerset);
```

Comment:
Again, this `.replace()` call is completely unnecessary here, the Set is already from the Hashmap and computed otherwise, adding it to the Set itself is enough, this is a wasteful call.